requirements-test.txt

```diff
@@ -1,7 +1,8 @@
-coverage==7.5.4
-pytest==8.2.2
+coverage==7.6.1
+pytest==8.3.2
 pytest-cov==5.0.0
 pytest-mock==3.14.0
-pytest-recording==0.13.1
+pytest-recording==0.13.2
 requests-mock==1.12.1
+ruff==0.6.1
```

Comment:
only one dep in the requirements-dev file so may as well just put it in with the test requirements